--- spdlog-1.12/CMakeLists.txt~	2016-08-26 14:39:00.000000000 +0200
+++ spdlog-1.12/CMakeLists.txt	2020-06-09 21:34:36.160954731 +0200
@@ -36,9 +36,9 @@
 # * https://github.com/forexample/package-example
 set(generated_dir "${CMAKE_CURRENT_BINARY_DIR}/generated")
 
-set(config_install_dir "lib/cmake/${PROJECT_NAME}")
+set(config_install_dir "lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}")
 set(include_install_dir "include")
-set(pkgconfig_install_dir "lib/pkgconfig")
+set(pkgconfig_install_dir "lib${LIB_SUFFIX}/pkgconfig")
 
 set(version_config "${generated_dir}/${PROJECT_NAME}ConfigVersion.cmake")
 set(project_config "${generated_dir}/${PROJECT_NAME}Config.cmake")
